orders/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
 
-from .views      import CartView
+from .views      import PaymentView,CartView
 
 urlpatterns = [
     path('/cart', CartView.as_view()),
+    path('/payment',PaymentView.as_view())
```

Comment:
```suggestion
    path('',PaymentView.as_view())
```
결국 주문과 관련된 기능을 하니까 8000/order 만으로 url이 구성되도 괜찮을 것 같습니다.

Reply:
수정 완료했습니다!